taxonomy-editor-frontend/src/pages/editentry/FetchAndDisplayRelations.jsx

```diff
@@ -0,0 +1,41 @@
+import useFetch from "../../components/useFetch";
+import { Box, Typography } from "@mui/material";
+import { Link } from "react-router-dom";
+
+const FetchRelations = ({url, title}) => {
+    const { data: relations, isPending, isError, isSuccess, errorMessage } = useFetch(url);
+    // Check error in fetch
+    if (isError) {
+        return (
+            <Box>
+                <Typography sx={{ml: 4}} variant='h5'>{errorMessage}</Typography>
+            </Box>
+        )
+    }
+    if (isPending) {
+        return (
+            <Typography sx={{ml: 4}} variant='h5'>Loading..</Typography>
+        )
+    }
+    return (
+        <Box>
+            <Typography sx={{ml: 4, mb: 1}} variant='h5'>{title}</Typography>
+
+            {/* When no parents or children are present */}
+            {relations && relations.length === 0 && <Typography sx={{ml: 8, mb: 1}} variant="h6"> None </Typography>}
+
+            {/* Renders parents or children of the node */}
+            {relations && relations.map(relation => (
+                <Box key={relation}>
+                    <Link to={`/entry/${relation}`} style={{color: '#0064c8', display: 'inline-block'}}>
```

Comment:
nit: could this color not be inlined? maybe include it in the `theme` of the `ThemeProvider` or something

Reply:
Ah, I don't think so since it's a react-router-dom component. So, I don't think there's a way to add it to the ThemeProvider.

Reply:
@aadarsh-ram instead of adding a style, you might be able to add a class corresponding to a CSS rule. (I maybe wrong, guessing out of blank :-D )

Reply:
@aadarsh-ram , please give a check to https://mui.com/material-ui/customization/color/